packages/fxa-content-server/app/styles/tailwind/inputs.css

```diff
@@ -2,8 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-module.exports = {
-  plugins: {
-    autoprefixer: {},
-  },
-};
+.input-text {
```

Comment:
I don't know why git thinks I renamed this lol. I removed `postcss.config.js` in `app/styles` because we had two config files (oops).

`input-text` isn't being used right now. I created it to begin TWifying the inputs, but ran into tooltip display issues, so it'll be punted to a follow up.